### DISCUSSION TITLE
[BUG] ERROR    module 'PySide2' has no attribute 'QtPrintSupport'

### DISCUSSION THREAD

**Discussion by crasyjoel** [dashingsoft/pyarmor#1181] [Q&A] [answered]
**When I use rft mode, it run with such error:**
(venv) PS C:\Users\xx\Documents\ita_final_v1_20230415> pyarmor gen -O dist --enable-rft C:\Users\xx\Documents\ita_final_v1_20230415\p\
INFO     Python 3.7.9
INFO     Pyarmor 8.1.7 (pro), 003169, non-profits
INFO     Platform windows.x86_64
INFO     search inputs ...
INFO     find package at C:\Users\LY\Documents\ita_final_v1_20230415\p
INFO     find 1 top resources
ERROR    module 'PySide2' has no attribute 'QtPrintSupport'

**PySide2 is sucessfully installed and program is packed&run well by v7.6.2.**

**Comment by jondy** [comment, marked as answer]
Please follow the guide in the documentation section `using rft mode` to fix this kind of problem.

rft mode is a complex feature, it need learn pyarmor to understand how rft mode work, and how to solve name error problems

please fix this kind of problem by users, I don't support this kind of bug report
